src/manage.c

```diff
@@ -497,9 +497,9 @@ static void manage_track()
 
     acq_status_t *acq = &acq_status[sid_to_index(ch->sid)];
 
-    /* Is ephemeris marked unhealthy? */
+    /* Is ephemeris or alert flag marked unhealthy?*/
     const ephemeris_t *e = ephemeris_get(ch->sid);
-    if (e->valid && !e->healthy) {
+    if (ch->nav_msg.alert || (e->valid && !e->healthy)) {
```

Comment:
Ah thanks for explaining that. I thought I needed something in the solution, but I wasn't sure after I saw this line!

I will try to figure out what is needed to change in there.